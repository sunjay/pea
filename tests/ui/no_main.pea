fn foo() {
    println!(3);
}
