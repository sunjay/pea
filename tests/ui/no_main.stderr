error: `main` function not found

error: aborting due to 1 previous error

